src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -155,7 +155,7 @@ public function normalize($object, $format = null, array $context = [])
 
                 if ($parameters = $this->getFiltersParameters($resourceClass, $operationName, $subResourceMetadata, $definitions, $serializerContext)) {
                     foreach ($parameters as $parameter) {
-                        if (!\in_array($parameter['name'], $parametersMemory, true)) {
+                        if (!\in_array($parameter['name'], $parametersMemory, true) && preg_match('/^[a-zA-Z0-9._$-]+$/', $parameter['name'])) {
```

Comment:
This regex really sucks, it doesn't allow array filters like `?foo[]=bar`, but this pattern is required by API Gateway on Swagger doc import